Source/WebCore/loader/appcache/ApplicationCacheStorage.cpp

```diff
@@ -1278,9 +1278,6 @@ bool ApplicationCacheStorage::writeDataToUniqueFileInDirectory(FragmentedSharedB
     
     do {
         path = makeString(FileSystem::encodeForFileName(createVersion4UUIDString()), fileExtension);
-        // Guard against the above function being called on a platform which does not implement
-        // createVersion4UUIDString().
-        ASSERT(!path.isEmpty());
```

Comment:
I suppose it’s good to remove this if it’s not needed. But why isn’t it needed any more?

Reply:
Because the above code was assuming that we have a platform which does not have appropriate `createVersion4UUIDString` function. But there is no such a platform in WebKit.